---
title: Generalizing RNA velocity to transient cell states through dynamical modeling
publication_types:
  - "2"
authors:
  - Volker Bergen
  - Marius Lange
  - Stefan Peidli
  - F Alexander Wolf
  - Fabian J Theis
doi: https://doi.org/10.1038/s41587-020-0591-3
publication: Nature Biotechnology
publication_short: Nat. Biotech.
abstract: RNA velocity has opened up new ways of studying cellular
  differentiation in single-cell RNA-sequencing data. It describes the rate of
  gene expression change for an individual gene at a given time point based on
  the ratio of its spliced and unspliced messenger RNA (mRNA). However, errors
  in velocity estimates arise if the central assumptions of a common splicing
  rate and the observation of the full splicing dynamics with steady-state mRNA
  levels are violated. Here we present scVelo, a method that overcomes these
  limitations by solving the full transcriptional dynamics of splicing kinetics
  using a likelihood-based dynamical model. This generalizes RNA velocity to
  systems with transient cell states, which are common in development and in
  response to perturbations. We apply scVelo to disentangling subpopulation
  kinetics in neurogenesis and pancreatic endocrinogenesis. We infer
  gene-specific rates of transcription, splicing and degradation, recover each
  cellâ€™s position in the underlying differentiation processes and detect
  putative driver genes. scVelo will facilitate the study of lineage decisions
  and gene regulation.
draft: false
featured: true
image:
  filename: featured
  focal_point: Smart
  preview_only: false
  caption: "scVelo generalized RNA velocity to transient populations through a
    dynamical model of splicing kinetics. "
date: 2022-06-06T15:25:00.526Z
---
