---
title: Concepts and limitations for learning developmental trajectories from
  single cell genomics
publication_types:
  - "2"
authors:
  - Sophie Tritschler
  - Maren BÃ¼ttner
  - David S Fischer
  - Marius Lange
  - Volker Bergen
  - Heiko Lickert
  - Fabian J Theis
doi: https://doi.org/10.1242/dev.170506
publication: Development
publication_short: Development
abstract: Single cell genomics has become a popular approach to uncover the
  cellular heterogeneity of progenitor and terminally differentiated cell types
  with great precision. This approach can also delineate lineage hierarchies and
  identify molecular programmes of cell-fate acquisition and segregation.
  Nowadays, tens of thousands of cells are routinely sequenced in single
  cell-based methods and even more are expected to be analysed in the future.
  However, interpretation of the resulting data is challenging and requires
  computational models at multiple levels of abstraction. In contrast to other
  applications of single cell sequencing, where clustering approaches dominate,
  developmental systems are generally modelled using continuous structures,
  trajectories and trees. These trajectory models carry the promise of
  elucidating mechanisms of development, disease and stimulation response at
  very high molecular resolution. However, their reliable analysis and
  biological interpretation requires an understanding of their underlying
  assumptions and limitations. Here, we review the basic concepts of such
  computational approaches and discuss the characteristics of developmental
  processes that can be learnt from trajectory models.
draft: false
featured: false
tags:
  - Review
  - Trajectory inference
  - Dynamical processes
image:
  filename: featured
  caption: "Trajectory inference enables the study of dynamical biological
  processes from cross-sectional snapshot data."
  focal_point: Smart
  preview_only: false
date: 2019-07-27T15:43:57.500Z
---
